st2common/st2common/bootstrap/runnersregistrar.py

```diff
@@ -305,6 +305,12 @@
                 'type': 'boolean',
                 'default': False
             },
+            'verify_ssl_cert': {
+                'description': 'Set to True to verify SSL certificate. Verification '
```

Comment:
On a related note - it would be good if we document SSL cert validation behavior in the runner docs - mention that we default to verifying the cert and we use requests CA bundle which comes from Mozilla...